django/core/validators.py

```diff
@@ -330,6 +329,10 @@ def __eq__(self, other):
             self.code == other.code
         )
 
+    @property
+    def limit_value(self):
+        return self._limit_value() if callable(self._limit_value) else self._limit_value
```

Comment:
Or make it a settable property then. I figured it wasn't common to change it in that manner, but it is easy to fix.

Regarding `__eq__`, I thought about that too, and it could just compare `_limit_value`.

I worry about other code added in future (or third-party code) not being aware that it needs to handle callables.

Reply:
In Django itself `limit_value` is used only in `IntegerField.validators` and due to the 3rd party apps we added a note in release notes and changed documentation so I would not worry about them.

We want to backport this fix to the `2.2` so IMO we should avoid unnecessary refactoring which may cause next regressions.

Reply:
Fair enough. I think the change I've suggested should be considered for master then as it'll be more robust in the long run.